src/styles/index.scss

```diff
@@ -1,3 +1,162 @@
+@import './variables';
+
+// Міксин для центрованих елементів
+@mixin flex-center($width, $height) {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $width;
+  height: $height;
+}
+
+@mixin star-position {
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+// Міксин для стилю тексту
+@mixin text-style($font-size, $font-weight, $color) {
+  font-size: $font-size;
+  font-weight: $font-weight;
+  color: $color;
+}
+
+html {
+  font-family: $font-family;
+}
+
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  width: 198px;
+  height: 406px;
+
+  &__content {
+    width: 198px;
+    height: 406px;
+    border-radius: 5px;
+  }
+
+  &__image {
+    @include flex-center(160px, 134px);
+
+    background-image: url(../images/imac.jpeg);
+    background-position: center;
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin-left: 19px;
+    margin-top: 32px;
+  }
+
+  &__title {
+    margin-top: 40px;
+    margin-left: $left-margin;
+    height: 36px;
+    width: 166px;
+
+    &-text {
+      @include text-style($text-size, 500, $text-color);
+
+      line-height: 18px;
+    }
+  }
+
+  &__code {
+    width: 98px;
+    height: 14px;
+    margin-top: 4px;
+    margin-left: $left-margin;
+    line-height: 14px;
+
+    &-text {
+      font-size: 10px;
+      font-weight: 400;
+      color: $subtext-color;
+    }
+  }
+
+  &__rating {
+    @include flex-center(166px, 16px);
+
+    width: 166px;
+    justify-content: space-between;
+    align-items: center;
+    margin-left: 17px;
+    margin-top: 16px;
+
+    // Цикл для зірок
+    @for $i from 1 through $stars-count {
```

Comment:
The loop for generating stars should be applied to a list of elements rather than applying styles to the `&-stars` class. Ensure that the HTML has multiple elements with the class `&__rating-star` to apply this loop correctly.